FinSearchUnified/Tests/Helper/StaticHelperTest.php

```diff
@@ -86,7 +94,18 @@ public static function configDataProvider()
                 'findologicDI' => false,
                 'isSearchPage' => false,
                 'isCategoryPage' => true,
+                'Request' => new RequestHttp(),
                 'expected' => false
+            ],
+            'FINDOLOGIC is active and Request is NULL' => [
```

Comment:
This should rather describe why the request is null e.g. `FINDOLOGIC is active but search was triggered via CLI`